README.md

```diff
@@ -15,13 +15,10 @@ A. Before you run the script, you will need to download your WordPress posts.
 2. On the left hand side, click on Tools - Export.
 3. Click on the Posts option, and select anything special you want.
 4. Click Download Export File.
-5. Save the Export File in a directory called "dowp" in your home directory (i.e. /Users/fred/dowp).
+5. Save the Export File as `wordpress.xml` in a directory called `dowp` in your home directory (i.e. `/Users/fred/dowp/wordpress.xml`).
```

Comment:
The default exported file has the blog name and date, which need to be renamed to `wordpress.xml` for the script.